src/main/java/dk/kb/image/util/ImageAccessValidation.java

```diff
@@ -0,0 +1,144 @@
+package dk.kb.image.util;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.StreamingOutput;
+
+import dk.kb.license.invoker.v1.ApiException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import dk.kb.image.config.ServiceConfig;
+import dk.kb.license.client.v1.DsLicenseApi;
+import dk.kb.license.model.v1.CheckAccessForIdsInputDto;
+import dk.kb.license.model.v1.CheckAccessForIdsOutputDto;
+import dk.kb.license.model.v1.UserObjAttributeDto;
+import dk.kb.license.util.DsLicenseClient;
+import dk.kb.util.Resolver;
+import dk.kb.util.webservice.exception.InternalServiceException;
+
+public class ImageAccessValidation {
+    private static final Logger log = LoggerFactory.getLogger(ImageAccessValidation.class);
+    private static DsLicenseApi licenseClient;
+    
+    public static enum ACCESS_TYPE {
+        ACCESS, NO_ACCESS, ID_NON_EXISTING
+    };
+
+    
+    
+    public static ACCESS_TYPE accessTypForImage(String resource_id) throws ApiException {
```

Comment:
JavaDoc please. And maybe mention that this resolves the access type for the role "everyone" and is likely to be changed in the future.

Also: You forgot an `e` in `accessTypForImage` :-)

Reply:
typoe fixed.

Reply:
Still no JavaDoc. Oh well...